src/compareNums.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+'use strict';
+
+const compareNumbers = (input, randomNum) => {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < input.length; i++) {
+    const isSameIndex = input[i] === randomNum[i];
+
+    if (isSameIndex) {
+      bulls++;
+    }
+
+    if (randomNum.includes(input[i]) && !isSameIndex) {
+      cows++;
+    }
```

Comment:
```suggestion
    bulls += isSameIndex;
    cows += randomNum.includes(input[i]) && !isSameIndex;
```

Boolean will be added as number (0 or 1)